tensorflow/python/framework/importer.py

```diff
@@ -232,232 +237,237 @@ def import_graph_def(graph_def, input_map=None, return_elements=None,
-                del node.attr[key]
-
-      output_types = _OutputTypes(node, op_dict)
-      name_to_op[node.name] = g.create_op(
-          node.op, [], output_types, name=node.name, attrs=node.attr,
-          compute_shapes=False, compute_device=False,
-          op_def=op_def)
-
-    # 2. Add inputs to the operations.
-    for node in graph_def.node:
-      op = name_to_op[node.name]
-      input_types = _InputTypes(node, op_dict)
-
-      # Rewrite the colocation attributes in the graph, since the
-      # names of new ops may have changed.
-      for key, value in op.node_def.attr.items():
-        if key == '_class':
-          class_values = value.list
-          new_class_values = []
-          for class_value in class_values.s:
-            if class_value.startswith(b'loc:@'):
-              op_to_bind_to = class_value[5:].decode()
-              # Find the op by its original name.
-              if op_to_bind_to not in name_to_op:
-                raise ValueError('Specified colocation to an op that '
-                                 'does not exist during import: %s in %s' % (
-                                     op_to_bind_to, node.name))
-              original_op = name_to_op[op_to_bind_to]
-              new_class_values.append(compat.as_bytes(
-                  'loc:@' + original_op.name))
-            else:
-              new_class_values.append(class_value)
-          value.list.CopyFrom(attr_value_pb2.AttrValue.ListValue(
-              s=new_class_values))
-
-      # NOTE(mrry): We cannot use zip here because control inputs do not appear
-      # in the list of input_types.
-      for i, input_name in enumerate(
-          [_CanonicalInputName(x) for x in node.input]):
-
-        if _IsControlInput(input_name):
-          # (a) Input is a control input that should be taken from an op
-          #     in "graph_def".
-          try:
-            source_op = name_to_op[input_name[1:]]
-          except KeyError:
-            raise ValueError(
-                _InvalidNodeMessage(
-                    node,
-                    'Control input %r not found in graph_def.' % (input_name,)))
-          # pylint: disable=protected-access
-          op._add_control_input(source_op)
-          # pylint: enable=protected-access
-
-        else:
-          try:
-            input_type = input_types[i]
-          except IndexError:
-            raise ValueError(_InvalidNodeMessage(
-                node, 'More inputs specified (%r) than the op expects.'
-                % (input_name,)))
-
-          if input_name in input_map:
-            # (b) Input should be replaced by a tensor from the caller.
-            source_tensor = input_map[input_name]
-            used_input_keys.add(input_name)
-
-          else:
-            # (c) Input should be taken from an op in `graph_def`.
-            operation_name, output_index = _ParseTensorName(input_name)
+  g = ops.get_default_graph() if graph is None else graph
```

Comment:
Actual new piece of code here, pretty much everything below this line (aside from `with g.as_default():`) is a diff caused by indentation and tweaks to keep the column width below 80.